cosmic-comp-config/src/lib.rs

```diff
@@ -8,12 +8,20 @@ pub mod input;
 pub mod workspace;
 
 #[derive(Clone, Debug, Default, PartialEq, CosmicConfigEntry)]
+#[version = 1]
 pub struct CosmicCompConfig {
     pub workspaces: workspace::WorkspaceConfig,
     pub input_default: input::InputConfig,
     pub input_touchpad: input::InputConfig,
     pub input_devices: HashMap<String, input::InputConfig>,
     pub xkb_config: XkbConfig,
+    /// Autotiling enabled
+    pub autotile: bool,
+    /// Tile windows on all workspaces
+    /// If set to false, autotile only applies to new windows
+    pub tile_all_windows: bool,
```

Comment:
I am not sure I understand how this works. Is this a toggle that moves all windows to the tiling layer at runtime? Does this change behaviour, if it is already set at cosmic-comp's launch?

Reply:
Ahh that is a good point. I completely forgot about static workspaces. On launch, this would set all workspaces to be the saved autotile value, but it should probably also include a list of static workspace names and their values, so they can be persisted.

Reply:
> Ahh that is a good point. I completely forgot about static workspaces.

We dropped static workspaces for the alpha, so don't worry about it now. We will need a list of those eventually, but that can be added later and is only relevant for cosmic-workspaces though.

Reply:
Ok, I will leave it out for now then. It should just be ok as an extra key  that can be added in the future.